aldryn_newsblog/locale/de/LC_MESSAGES/django.po

```diff
@@ -63,360 +65,360 @@ msgstr ""
 #: templates/aldryn_newsblog/plugins/article_search.html:10
 msgid "Go"
-msgstr ""
+msgstr "Los"
 
 #: boilerplates/bootstrap3/templates/aldryn_newsblog/plugins/categories.html:6
 #: boilerplates/bootstrap3/templates/aldryn_newsblog/plugins/tags.html:6
 #: templates/aldryn_newsblog/plugins/categories.html:5
 #: templates/aldryn_newsblog/plugins/tags.html:5
 msgid "All"
-msgstr ""
+msgstr "Alle"
 
 #: boilerplates/bootstrap3/templates/aldryn_newsblog/plugins/related_articles.html:9
 #: templates/aldryn_newsblog/plugins/related_articles.html:8
 msgid "by"
-msgstr ""
+msgstr "von"
 
 #: cms_app.py:15
 msgid "NewsBlog"
-msgstr ""
+msgstr "News Blog"
 
 #: cms_appconfig.py:19
 msgid "the-eagle-has-landed/"
-msgstr ""
+msgstr "der-adler-ist-gelandet/"
 
 #: cms_appconfig.py:20
 msgid "1969/the-eagle-has-landed/"
-msgstr ""
+msgstr "1969/der-adler-ist-gelandet/"
 
 #: cms_appconfig.py:21
 msgid "1969/07/the-eagle-has-landed/"
-msgstr ""
+msgstr "1969/07/der-adler-ist-gelandet/"
 
 #: cms_appconfig.py:22
 msgid "1969/07/16/the-eagle-has-landed/"
-msgstr ""
+msgstr "1969/07/16/der-adler-ist-gelandet/"
 
 #: cms_appconfig.py:23
 msgid "1969/07/16/11/"
-msgstr ""
+msgstr "1969/07/16/11/"
 
 #: cms_appconfig.py:27
 msgid "Allow"
-msgstr ""
+msgstr "Erlauben"
 
 #: cms_appconfig.py:28
 msgid "Redirect to permalink (default)"
-msgstr ""
+msgstr "Zum Permalink weiterleiten (Standard)"
 
 #: cms_appconfig.py:29
 msgid "Permanent redirect to permalink"
-msgstr ""
+msgstr "Permanente Weiterleitung zum Permalink"
 
 #: cms_appconfig.py:30
 msgid "Return 404: Not Found"
-msgstr ""
+msgstr "Zeige 404: Nicht gefunden"
 
 #: cms_appconfig.py:42
 msgid "application title"
-msgstr ""
+msgstr "Applikationstitel"
 
 #: cms_appconfig.py:45
 msgid "permalink type"
-msgstr ""
+msgstr "Permalink Typ"
 
 #: cms_appconfig.py:47
 msgid ""
 "Choose the style of urls to use from the examples. (Note, all types are "
 "relative to apphook)"
-msgstr ""
+msgstr "Stil der URLs aus den Beispielen auswählen. (Notiz, alle Typen sind relativ zur Applikations Seite)"
 
 #: cms_appconfig.py:51
 msgid "non-permalink handling"
-msgstr ""
+msgstr "Nicht-Permanente Links Handhabung"
 
 #: cms_appconfig.py:54
 msgid "How to handle non-permalink urls?"
-msgstr ""
+msgstr "Wie sollen nicht-Permamente Links gehandhabt werden?"
 
 #: cms_appconfig.py:58
 msgid "Paginate size"
-msgstr ""
+msgstr "Anzahl der Seitennummern"
 
 #: cms_appconfig.py:61
 msgid "When paginating list views, how many articles per page?"
-msgstr ""
+msgstr "Wie viele Artikel sollen pro Seite dargestellt werden?"
 
 #: cms_appconfig.py:68
 msgid "Prefix for template dirs"
-msgstr ""
+msgstr "Präfix für Template \"dirs\""
 
 #: cms_appconfig.py:72
 msgid "Auto-create authors?"
-msgstr ""
+msgstr "Auto-Kreierte Auhore?"
 
 #: cms_appconfig.py:74
 msgid "Automatically create authors from logged-in user?"
-msgstr ""
+msgstr "Kreiere automatisch Autoren von eingeloggten Benutzern?"
```

Comment:
``Sollen automatisch Autoren für eingeloggte Benutzer erstellt werden?``

Need a better translation for ``eingeloggte``.